array-methods.js

```diff
@@ -25,8 +25,8 @@ var sumOfBankBalances = dataset.bankBalances.reduce(function(accumulator, curren
  */
 
 var sumOfInterests = dataset.bankBalances.reduce(function(accumulator, current){
-  if (current.state === 'WI' || current.state === 'IL' || current.state === 'WY' || current.state === 'OH' || current.state === 'GA' || current.state === 'DE'){
-    return accumulator + Math.round((((current.amount * 1) * 0.189))) ; 
+  if (current.state.includes('WI') || current.state.includes('IL') || current.state.includes('WY') || current.state.includes('OH') || current.state.includes('GA') || current.state.includes('DE')){
```

Comment:
This could be rewritten as `['WI', 'IL', 'WY', 'OH', 'GA', 'DE'].includes(current.state)` to make the whole if statement simpler. This goes the same for all the other state checks.